validate_sptx/schema/experiment.json

```diff
@@ -14,10 +14,8 @@
       "$ref": "schema/version.json"
     },
     "primary_images": {
-      "type": "string",
-      "description": "Images that measure transcript or protein targets in one or more fields of view.",
-      "example": "primary_images.json",
-      "pattern": "^.*?.json$"
+      "$ref": "schema/fov_manifest.json",
```

Comment:
This still does require a match of `"pattern": "^.*?.json$"`, does it not? It's just that it needs to be more specific? 

And, shouldn't the description string refer to a manifest? 

`"description": "Manifest specifying the location(s) of one or more Field of View sub-manifests"`

Also I don't see code here to make things more strict, yet.

Reply:
I have as yet not found a way to do what I think you and @ttung actually want which is to recursively validate _linked files_. For the moment, I've just dropped the `possible_manifests` logic out of the validator. The next step would be to have validator discover (or parse out of the schema) what to expect on a file by file basis, but I don't think anything is built into jsonschema for that. The alternative would be to go to monolithic schemas & instance documents.